testsuite/integration-arquillian/tests/base/src/test/resources/authorization-test/enforcer-config-claims-provider.json

```diff
@@ -0,0 +1,81 @@
+{
+  "realm": "test-realm-authz",
+  "realm-public-key": "MIGfMA0GCSqGSIb3DQEBAQUAA4GNADCBiQKBgQCrVrCuTtArbgaZzL1hvh0xtL5mc7o0NqPVnYXkLvgcwiC3BjLGw1tGEGoJaXDuSaRllobm53JBhjx33UNv+5z/UMG4kytBWxheNVKnL6GgqlNabMaFfPLPCF8kAgKnsi79NMo+n6KnSY8YeUmec/p2vjO2NjsSAVcWEQMVhJ31LwIDAQAB",
+  "auth-server-url": "http://localhost:8180/auth",
+  "ssl-required": "external",
+  "resource": "test-app-authz",
+  "bearer-only": true,
+  "credentials": {
+    "secret": "secret"
+  },
+  "policy-enforcer": {
+    "paths": [
+      {
+        "path": "/claims-provider",
+        "methods": [
+          {
+            "method": "POST",
+            "scopes": [
+              "create"
+            ]
+          }
+        ],
+        "claim-information-point": {
+          "claims": {
+            "claim-from-request-parameter": "{request.parameter['a']}",
+            "claim-from-header": "{request.header['b']}",
+            "claim-from-cookie": "{request.cookie['c']}",
+            "claim-from-remoteAddr": "{request.remoteAddr}",
+            "claim-from-method": "{request.method}",
+            "claim-from-uri": "{request.uri}",
+            "claim-from-relativePath": "{request.relativePath}",
+            "claim-from-secure": "{request.secure}",
+            "claim-from-json-body-object": "{request.body['/a/b/c']}",
+            "claim-from-json-body-array": "{request.body['/d/1']}",
+            "claim-from-body": "{request.body}",
+            "claim-from-static-value": "static value",
+            "claim-from-multiple-static-value": ["static", "value"]
+          }
+        }
+      },
+      {
+        "path": "/http-post-claim-provider",
+        "claim-information-point": {
+          "http": {
+            "url": "http://localhost:8989/post-claim-information-provider",
+            "method": "POST",
+            "headers": {
+              "Content-Type": "application/x-www-form-urlencoded",
+              "header-b": ["header-b-value1", "header-b-value2"],
+              "Authorization": "Bearer {keycloak.access_token}"
+            },
+            "parameters": {
+              "param-a": ["param-a-value1", "param-a-value2"],
+              "param-subject": "{keycloak.subject}",
+              "param-user-name": "{keycloak.preferred_username}"
+            }
```

Comment:
unlike the simple claims above, I don't see an option to tell by configuration which part of the http response should be mapped to a claim

Reply:
The idea behind the "http" CIP provider is that the endpoint you are calling should return a JSON response and every single claim from the response will be mapped as a claim.

Reply:
mapping all the claims from the JSON response into PermissionRequest claims is an overkill and will have bad performance. 
The JSON in http response might be large while usually one needs only a single attribute as a claim. 
Therefore, the better approach is to use a JSON path format (RFC 6901) to parse the response.Just like it was done with simple claims mapped from the HTTP request directly ("/a/b/1" etc.)

Reply:
I partially agree with you, so you still have the possibility to map every single claim from the external http service response.

Regarding your suggestion, you can now provide a `claims` configuration where the key is the claim name and the value is a json path from where the value should be obtained. See https://github.com/keycloak/keycloak/pull/5144/files#diff-1703da928fbf48717884b3d5543851cfR46.